demo/src/app/components/+tooltip/demos/container/container.html

```diff
@@ -1,3 +1,4 @@
+<p>Be careful! If you add <code>overflow: hidden</code> property to any parent block of the tooltip it can looks like in the first button of this section</p>
```

Comment:
Be careful! If you add <code>overflow: hidden</code> property to any parent block of the tooltip it can look cropped. Hover over the first button in this section to see the example.